scripts/qa_crowbarsetup.sh

```diff
@@ -1248,34 +1248,41 @@ function set_node_role_and_platform()
     rm -f $t
 }
 
+# Reboot the nodes with ipmi
+function do_ipmi()
```

Comment:
this function name is too generlc, the name should say what the function is doing
maybe something ike `reboot_nodes_via_ipmi`